frontend/wallet/src/render/components/Button.re

```diff
@@ -93,7 +93,10 @@ module Styles = {
             color(hoverColor(mode)),
             cursor(`pointer),
           ]),
-          focus([backgroundColor(hoverBgColor(mode))]),
+          focus([
+            backgroundColor(hoverBgColor(mode)),
+            color(hoverColor(mode)),
+          ]),
           active([backgroundColor(hoverBgColor(mode))]),
```

Comment:
Do we need it on the active style too? Not sure I understand what's going on here

Reply:
not sure, the bug came from when the user clicks cancel on the modal and so i was thinking that the button was still in focus at the time, but i think active refers to when the user has already clicked on it and thus the modal is blocking the button so you can't really see the change.. i can add it there though for consistency

Reply:
I'd imagine not all buttons have a modal that pops in front of them